src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/form_help.html.php

```diff
@@ -1,3 +1,4 @@
 <?php if (!empty($help)): ?>
-    <p id="<?php echo $view->escape($id); ?>_help" class="help-text"><?php echo $view->escape(false !== $translation_domain ? $view['translator']->trans($help, array(), $translation_domain) : $help); ?></p>
+    <?php $help_attr['class'] = trim((isset($help_attr['class']) ? $help_attr['class'].' help-text' : 'help-text')); ?>
```

Comment:
you could apply the trimming around the concatenation only, as the other alternative is already trimmed.